specification/automation/resource-manager/Microsoft.Automation/stable/2021-06-22/examples/hybridRunbookWorkerGroupGet.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg",
+    "automationAccountName": "testaccount",
+    "hybridRunbookWorkerGroupName": "TestHybridGroup",
+    "api-version": "2020-01-13-preview"
+  },
```

Comment:
Update API version in this example

Reply:
This file is not present now, this is from initial commit created for base. Current examples are using correct api-version.